modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/GodotMemberData.cs

```diff
@@ -6,20 +6,18 @@ namespace Godot.SourceGenerators
     public readonly struct GodotMethodData
     {
         public GodotMethodData(IMethodSymbol method, ImmutableArray<MarshalType> paramTypes,
-            ImmutableArray<ITypeSymbol> paramTypeSymbols, MarshalType? retType, ITypeSymbol? retSymbol)
+            ImmutableArray<ITypeSymbol> paramTypeSymbols, (MarshalType marshalType, ITypeSymbol typeSymbol)? retType)
```

Comment:
.NET naming guidelines recommend using PascalCase for tuple items.

```suggestion
            ImmutableArray<ITypeSymbol> paramTypeSymbols, (MarshalType MarshalType, ITypeSymbol TypeSymbol)? retType)
```